packages/common/src/permissions/types/IPermissionPolicy.ts

```diff
@@ -54,6 +54,8 @@ export interface IPermissionPolicy {
    * More complex policies can be defined with a set of assertions
    */
   assertions?: IPolicyAssertion[];
+
+  features?: string[];
```

Comment:
IPermissionPolicy now has a features array that defines the features that must be enabled in order for the permission to be granted.